code/go/0chain.net/miner/protocol_block.go

```diff
@@ -776,7 +776,9 @@ func (mc *Chain) generateBlock(ctx context.Context, b *block.Block,
 		}
 	}
 
-	if config.DevConfiguration.IsBlockRewards {
+	if config.DevConfiguration.IsBlockRewards &&
+		b.Round%config.SmartContractConfig.GetInt64("smart_contracts.storagesc.block_reward.trigger_period") == 0 {
```

Comment:
Get config from MPT, anyway, should avoid read from file directly.

Reply:
to be done later